apps/gateway-ui/src/components/federations/ConnectFederation.tsx

```diff
@@ -151,15 +108,15 @@ export const ConnectFederation = React.memo(function ConnectFederation({
             />
             <Button
               borderRadius='8px'
-              maxW='210px'
```

Comment:
Note for reviewers: We're removing this so that the `loadingText` below isn't cut off